### ISSUE TITLE
melt curve analysis

### ISSUE THREAD

**Issue by ShahimEssaid** [ShahimEssaid/gcissues#6]
_From [vasil...@ohsu.edu](https://code.google.com/u/108803237899917466626/) on December 12, 2012 16:23:16_

\<a href="http://jaxmice.jax.org/protocolsdb/f?p=116:2:912662273403085::NO:2:P2_MASTER_PROTOCOL_ID,P2_JRS_CODE:1987,009602" rel="nofollow">http://jaxmice.jax.org/protocolsdb/f?p=116:2:912662273403085::NO:2:P2_MASTER_PROTOCOL_ID,P2_JRS_CODE:1987,009602</a>

_Original issue: http://code.google.com/p/eagle-i/issues/detail?id=175_

**Comment by ShahimEssaid** [comment]
_From [sc...@eagle-i.org](https://code.google.com/u/105108392056921798412/) on March 22, 2013 10:57:34_

**Status:** Started  
  **Owner:** sc...@eagle-i.org  
  **Labels:** technique

**Comment by ShahimEssaid** [comment]
_From [sc...@eagle-i.org](https://code.google.com/u/105108392056921798412/) on March 22, 2013 15:54:15_

Added to ontology.

**Status:** Fixed
